File: squad_asc_Rawan_guessinggame.c
#include <stdio.h>

void main()
{
    int secretNumber = 7;
    int guess, counter=0;
    printf("You have 3 guesses \nEnter a number: ");
    scanf("%d",&guess);
    while(guess!= secretNumber && counter <2)
    {
        printf("Guess another number: ");
        scanf("%d",&guess);
        counter ++;
    }
    if (guess ==secretNumber)
        printf("CONGRATULATIONS, YOU WIN !");
    else
        printf("Out of guesses");
}

File: squad_asc_Rawan_guessinggame.py
secret_word = "sea"
guess =""
count =3
while guess != secret_word and count != 0:
    print("You have",count,"guesses")
    guess = input("Enter a word: ")
    count -=1
if guess == secret_word: 
    print("CONGRATULATIONS, YOU WIN !")
else:
     print("Out of guesses, YOU LOSE!")    